monkey/tests/unit_tests/common/agent_configuration/test_agent_configuration.py

```diff
@@ -258,49 +214,20 @@ def test_agent_configuration():
     assert isinstance(config.credential_collectors[0], PluginConfiguration)
     assert isinstance(config.payloads[0], PluginConfiguration)
     assert isinstance(config.propagation, PropagationConfiguration)
-    assert json.loads(config_json) == AGENT_CONFIGURATION
+    assert config_dict == convert_lists_to_tuples(AGENT_CONFIGURATION.copy())
 
 
 def test_agent_configuration__negative_keep_tunnel_open_time():
     negative_keep_tunnel_open_time_configuration = AGENT_CONFIGURATION.copy()
     negative_keep_tunnel_open_time_configuration["keep_tunnel_open_time"] = -1
 
-    with pytest.raises(InvalidConfigurationError):
-        AgentConfiguration.from_mapping(negative_keep_tunnel_open_time_configuration)
+    with pytest.raises(ValueError):
+        AgentConfiguration(**negative_keep_tunnel_open_time_configuration)
 
 
 def test_incorrect_type():
-    valid_config = AgentConfiguration.from_mapping(AGENT_CONFIGURATION)
-    with pytest.raises(InvalidConfigurationError):
+    valid_config = AgentConfiguration(**AGENT_CONFIGURATION)
+    with pytest.raises(TypeError):
         valid_config_dict = valid_config.__dict__
         valid_config_dict["keep_tunnel_open_time"] = "not_a_float"
         AgentConfiguration(**valid_config_dict)
-
-
-def test_to_from_mapping():
-    config = AgentConfiguration.from_mapping(AGENT_CONFIGURATION)
-
-    assert AgentConfiguration.to_mapping(config) == AGENT_CONFIGURATION
-
-
-def test_from_mapping__invalid_data():
-    dict_ = deepcopy(AGENT_CONFIGURATION)
-    dict_["payloads"] = "payloads"
-
-    with pytest.raises(InvalidConfigurationError):
-        AgentConfiguration.from_mapping(dict_)
-
-
-def test_to_from_json():
```

Comment:
We still want to make sure we can jsonify agent configuration?

Reply:
[This](https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeljson) exists so I don't think we need to.